src/app/modules/convai-checker/perspective-status.component.ts

```diff
@@ -145,7 +145,7 @@ export class PerspectiveStatus implements OnChanges, AfterViewInit, AfterViewChe
     "#layer1", "#layer2", "#layer3"
   ];
 
-  private showFeedbackQuestion: boolean = false;
+  public showFeedbackQuestion: boolean = false;
```

Comment:
Something I learned today: the public keyword is redundant, things are public by default. You can just remove private and it will have the same effect.

Reply:
That's actually not true in Typescript compilation; private methods can be inlined and optimized in various ways, but public one's cannot. When things get uglified, private methods get renamed, but public ones don't. If you secretly depend on a private method (as you do above) in  an interface, then compilation methods will break your code. Luckily the new library builder notices these things and gives warnings (which is how I found this).

Reply:
I didn't know the bit about the compilation, that's interesting. The internal Typescript style guide strongly discourages the use of the public keyword outside of the constructor parameters (the reason they claim is that it doesn't make a difference from the default of not including it). I wonder if there's something different about internal compilation, I'll have to look into that!